### ISSUE TITLE
Can't create fifo queue

### ISSUE THREAD

**Issue by zkanda** [localstack/localstack#335]
Can't create fifo queue on either `latest` or `0.8.0`,

I'm trying this in my local: 
* `aws --endpoint-url=http://localhost:4576 sqs create-queue --queue-name "test-queue.fifo" --attributes "FifoQueue=true"`

It's giving me this error:

```
An error occurred (InvalidParameterValue) when calling the CreateQueue operation: InvalidParameterValue; see the SQS docs.
```

Am I missing something?

Edit: I tried it on the old  atlassianlabs/localstack:0.6.2 and it works, but a lot of functionality is broken, like listing queues.

**Comment by whummer** [comment]
Thanks for reporting @zkanda . Looks like this is an upstream issue with ElasticMQ, where FIFO queues are not supported yet (https://github.com/adamw/elasticmq/issues/92).

Are you relying on the FIFO functionality, in terms of delivery guarantees etc? If not, then we could try to push a temporary hack just to make this work with "*.fifo" queue name suffixes. Although the proper way would be to create a pull request with ElastichMQ, of course.

**Comment by zkanda** [comment]
@whummer not relying on the delivery guarantees for FIFO, for now a `.fifo` would be nice so we can use the latest. :+1:

**Comment by whummer** [comment]
This should be fixed in #361 . Can you please confirm that it works with the latest version @zkanda

**Comment by zkanda** [comment]
I can confirm that this is now working as expected, thank you so much. 👍
